agent.py

```
@@ -111,12 +118,16 @@ async def _analyze_messages(self, messages: RequestAnalyseVoiceCall):
-                json=messages.model_dump(mode="json"),
-                headers={"Authorization": f"ApiKey {settings.kai_api_secret_key}"},
-            )
-            response.raise_for_status()
+            try:
+                response = await client.post(
+                    f"{settings.kai_api_base_url}/kai/voice-call/{self.metadata.voice_call_id}/analyze/",
+                    json=messages.model_dump(mode="json"),
+                    headers={"Authorization": f"ApiKey {settings.kai_api_secret_key}"},
+                )
+                response.raise_for_status()
+            except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)